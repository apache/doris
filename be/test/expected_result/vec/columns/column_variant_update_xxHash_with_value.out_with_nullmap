column: variant with hashes: 9301615508800336962 with ptr: 2225