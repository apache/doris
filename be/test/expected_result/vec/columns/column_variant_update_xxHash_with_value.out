column: variant with hashes: 1417894694298673530 with ptr: 2023